main/res/values/strings.xml

```diff
@@ -1287,6 +1287,7 @@
     <string name="map_as_list">Show as list</string>
     <string name="map_select_multiple_items">Select item</string>
     <string name="map_routing">Routing</string>
+    <string name="map_routing_brouter">Routing is only available when <a href="">brouter</a> is installed</string>
```

Comment:
I'm not that good at English, is "when" correct or should it rather be "if"?

Reply:
yes, IMHO it should be "if"